src/mattack_actors.cpp

```diff
@@ -360,8 +360,10 @@ void bite_actor::load_internal( const JsonObject &obj, const std::string &src )
 void bite_actor::on_damage( monster &z, Creature &target, dealt_damage_instance &dealt ) const
 {
     melee_actor::on_damage( z, target, dealt );
-    if( target.has_effect( effect_grabbed ) && one_in( no_infection_chance - dealt.total_damage() ) ) {
-        const bodypart_id &hit = dealt.bp_hit;
+    const bodypart_id &hit = dealt.bp_hit;
+    if( target.has_effect( effect_grabbed ) && one_in( no_infection_chance - dealt.total_damage() ) &&
+        ( ( target.is_avatar() || target.is_npc() ) &&
```

Comment:
I think monsters don't get grabbed so I'm not sure this check is necessary

Reply:
Just checked mattack::grab() in monattack.cpp ( line 2750 ) and didn't see codes about only grab characters... Maybe I overlooked something ( But this check exists before the PR )